bin/fixmystreet.com/generate_council_location

```diff
@@ -1,11 +1,24 @@
 #!/usr/bin/perl
 
+=head1 USAGE
+
+bin/fixmystreet.com/generate_council_location <mapit_area_id>
+
+=head1 DESCRIPTION
+
+Generates a location hash for a council, which can be used in the cobrand's
+disambiguate_location method.
+
+=cut
+
 use strict;
 use JSON;
 use LWP::Simple;
 
 my $id = shift;
 
+die "Usage: $0 <mapit_area_id>\n" unless $id;
+
 my $url = sprintf( 'http://mapit.mysociety.org/area/%s/geometry', $id);
```

Comment:
While we're here, switching to https would speed it up a tiny bit ;)

Reply:
Done in 40655d7d4d744aaf2bf9a1fe40573cc5af5ea567